board/can_definitions.h

```diff
@@ -15,6 +15,7 @@ const uint8_t PANDA_BUS_CNT = 4U;
   #define CANPACKET_DATA_SIZE_MAX 8U
 #endif
 
+// cppcheck-suppress [misra-c2012-2.3,misra-c2012-2.4] ; False Positive
```

Comment:
can you make an issue for this on cppcheck's issue tracker?

Reply:
After further introspection I realized it wasn't a false positive. My apologies for the trouble.